python/lsst/dax/apdb/cassandra/apdbCassandraSchema.py

```diff
@@ -360,6 +360,39 @@ def empty(self) -> bool:
                 f"Only some required APDB tables exist: {existing_tables}, missing tables: {missing_tables}"
             )
 
+    def existing_tables(self, *args: ApdbTables) -> dict[ApdbTables, list[str]]:
+        """Return the list of existing table names for given table.
+
+        Parameters
+        ----------
+        *args : `ApdbTables`
+            Tables for which to return their existing table names.
+
+        Returns
+        -------
+        tables : `dict` [`ApdbTables`, `list`[`str`]]
+            Mapping of the APDB table to the list of the existing table names.
+            More than one name can be present in the list if configuration
+            specifies per-partition tables.
+        """
+        if self._time_partition_tables and not set(args).isdisjoint(self._time_partitioned_tables):
```

Comment:
`not isdisjoint` isn't not difficult to understand. Maybe use `intersection` instead?

Reply:
`intersection` creates new `set`, I do not like making complex structure to only test that it's not empty.

Reply:
A set is hardly a "complex" structure -- it's one of the fundamental collection types. Are you really saving enough computer time to be worth the human time?

Reply:
A set is fundamental, but it is definitely much more complex than a simple boolean. I do care about computer time.